app/components/public/session-item.hbs

```diff
@@ -48,7 +48,7 @@
           {{/if}}
         </div>
       {{else}}
-        <div class="left floated eleven wide column">
+        <div class="left floated eleven wide column" style="padding-bottom: 0 !important;" >
           <div class="d-flex wrap">
             {{#each @session.speakers as |speaker|}}
               <div class="d-flex items-center speaker-card pr-2 pb-4">
```

Comment:
plz make pb-0 and define it.

Reply:
no, it will become clumsy when there are multiple speakers